setup.py

```diff
@@ -26,5 +26,6 @@
     install_requires=[
         'Flask>=0.10',
         'six==1.9.0',
+        'requests==2.7.0',
```

Comment:
This project is a python package. In order to be installable by pip it must have a `setup.py` which tells pip how to install it (including it's dependencies). What we could do is have `setup.py` read the requirements from a `requirements.txt` file. Ultimately this is a different kind of thing to the other repos though.